pkg/kb-summarizer/summarizer/summarizer.go

```diff
@@ -249,6 +249,9 @@ func (s *Summarizer) processOneResultFileForHost(results *kb.Controls, hostname
 				continue
 			}
 
+			if check.Type == CheckTypeSkip {
+				check.State = NA
+			}
 			if msg, ok := s.notApplicable[check.ID]; ok {
```

Comment:
Guess, we can remove `s.notApplicable` map altogether?

Reply:
We can, but maybe we need not change the summarizer tool and it can take in the input maps as input - there will not be the defaultSkip and Notapplicable maps provided when Rancher invokes this summarizer tool. But some other cli/client can still run the summarizer by providing these maps. Does that sound ok? @leodotcloud